src/main/java/org/apache/commons/configuration2/AbstractConfiguration.java

```diff
@@ -419,6 +419,20 @@ public final boolean containsKey(final String key) {
         }
     }
 
+    /**
+     * {@inheritDoc} This implementation handles synchronization and delegates to {@code containsKeyInternal()}.
+     */
+    @Override
+    public final boolean containsValue(final String value) {
+        Objects.requireNonNull(value, "Parameter \"value\" cannot be null");
```

Comment:
I agree, that makes sense, since `containsValue` delegates to containsValueInternal there could be an implementation that allows null values.

Will remove.

Reply:
If every configuration can have a different implementation for containsValue it means that every configuration must be tested for their (existing) implementation of containsValue.

Is it ok if I do it all in this PR?